packages/react/src/components/Table/Table.jsx

```diff
@@ -253,6 +255,7 @@ const propTypes = {
       onColumnResize: PropTypes.func,
       onOverflowItemClicked: PropTypes.func,
       onTableErrorStateAction: PropTypes.func,
+      onToggleAggregations: PropTypes.func,
```

Comment:
I went back and forth on this since the aggregations are in the table, but I since the action is fired from the toolbar, this makes the most sense. Updated.